apps/ledger-live-mobile/src/components/CustomImageDeviceAction.tsx

```diff
@@ -116,50 +103,21 @@ const CustomImageDeviceAction: React.FC<Props & { remountMe: () => void }> = ({
       : "Error: " + error.name
     : undefined;
 
+  const extraErrorArgs = {
+    deviceName: device.deviceName || "",
+    modelId: device.modelId,
+  };
+
   return (
     <ImageSourceContext.Provider value={{ source }}>
       {trackScreenName ? <TrackScreen category={trackScreenName} /> : null}
       <Flex flexDirection="column" flex={1} alignSelf="stretch">
         {isError ? (
-          <Flex flex={1}>
-            {renderError({
-              t,
-              error,
-              device,
-              ...(isRefusedOnStaxError
-                ? { Icon: Icons.CircledAlertMedium, iconColor: "warning.c100" }
-                : {}),
-            })}
-            {}
-            <Button
-              size="large"
-              type="main"
-              outline={false}
-              onPress={handleRetry}
-              event="button_clicked"
-              eventProperties={
-                isRefusedOnStaxError
-                  ? analyticsRefusedOnStaxUploadAnotherEventProps
-                  : analyticsErrorTryAgainEventProps
-              }
-            >
-              {isRefusedOnStaxError
-                ? t("customImage.uploadAnotherImage")
-                : t("common.retry")}
-            </Button>
-            {isRefusedOnStaxError ? (
-              <Flex py={7}>
-                <Link
```

Comment:
just writing this here so we don't forget to put back a "skip" button somehow once the discussion with product/design has advanced

Reply:
Likely we could have a conditional secondary action that is shown only when it receives the prop (like we discussed), it should be easy to implement and would be compatible with a device action version that doesn't have it.